typeclasses/characters.py

```diff
@@ -101,6 +150,10 @@ def become_race(self, race):
         Returns:
             bool: True if successful, False if otherwise
         """
+        # check if race is a valid races object
+        if not isinstance(race, races):
```

Comment:
This should be removed, as it will break `become_race`. `load_race` takes care of all the logic you would want here, and the `race` being passed in will be a string, so this would fail anyhow.

In addition, `races` is a module, `races.Race` would be the class to hit `isinstance` on.

However, this should just be removed :)